### PR TITLE
Update IDL to program v3.1.0

### PR DESCRIPTION
**Pull request by Orelsanpls**

### CODE REVIEW

**Comment by acamill** [comment]
@cnek did we do this back then? Wondering if we got the program right and client wrong or vice versa here

**Comment by cnek** [comment]
> @cnek did we do this back then? Wondering if we got the program right and client wrong or vice versa here

https://github.com/UXDProtocol/uxd-client/pull/13
we merged this "redeem only mode" but actually this is excluded in v3.1.0
anyway, `disableDepositoryRegularMinting` is what we need now

File: src/idl.ts [hunk 1]
```diff
@@ -986,15 +986,15 @@ export const IDL: Idl = {
...
+          name: 'disable',
           type: 'bool',
         },
       ],
     },
   ],
   accounts: [
     {
-      name: 'controller',
+      name: 'Controller',
```
Comment (cnek) [inline src/idl.ts]: this has to be in camel case iirc

File: src/idl.ts [hunk 2]
```diff
@@ -1052,7 +1052,7 @@ export const IDL: Idl = {
       },
     },
     {
-      name: 'mangoDepository',
+      name: 'MangoDepository',
```
Comment (cnek) [inline src/idl.ts]: same

File: src/idl.ts (hunk 1 above)
Comment (acamill) [reply src/idl.ts]: Yes not sure why it changed

Comment (acamill) [reply src/idl.ts]: You must take these from the type.ts file, and the second half of the file (there are two parts)

**Review by acamill — APPROVED** [review]

**Review by ken-chow-crypto — APPROVED** [review]

**Review by cnek — APPROVED** [review]

**Review by acamill — APPROVED** [review]
LGTM

**Pull request merged**